### PR TITLE
Fix titles in test, broadcast and interact with network pages in SDK docs - closes #540

### LINKED ISSUE #540: Fix titles in test, broadcast & interact with network pages in SDK docs
Test transaction types: Plural, why not singular? Test a custom transaction
 Interact with the network > Interact with the API
 Broadcast transactions > Broadcast a transaction. Should it be before Interact with the network in the navigation?

### PR DESCRIPTION
**Pull request by chris529**
closes #540 

Parent issue #535

### CODE REVIEW

**Review by Tschakki — CHANGES_REQUESTED** [review]
Why are you fixing Hello World here? Isn't this already done in #539 ?

**Comment by chris529** [comment]
Hi Mona, Apologies this was an error...You are correct 'Hello World' was done in #539

**Review by chris529 — COMMENTED** [review]
Looks good..

File: modules/ROOT/pages/guides/index.adoc [hunk 1]
```diff
@@ -26,17 +26,17 @@ The most important parts of building a blockchain application are described belo
...
-== The Hello world app
+== The Hello World app
 
-As an example, the guides use the {url_github_hello}[Hello world] app, which contains a very basic blockchain application with one custom transaction type and a simple frontend.
+As an example, the guides use the {url_github_hello}[Hello World] app, which contains a very basic blockchain application with one custom transaction type and a simple frontend.
 
 This basic example describes a minimal set of code which is required to implement a blockchain application.
 
-If you wish to follow the guides on a 1-to-1 basis, it is recommend to set up the 'Hello world' app as described below.
+If you wish to follow the guides on a 1-to-1 basis, it is recommend to set up the 'Hello World' app as described below.
```
Comment (chris529) [inline modules/ROOT/pages/guides/index.adoc]:
```suggestion
If you wish to follow the guides on a 1-to-1 basis, it is recommended to set up the 'Hello World' app as described below.
```

**Review by chris529 — COMMENTED** [review]
Only 1 minor point :-)

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please check for occurrences of the old titles in the other pages, and update them as well

**Comment by chris529** [comment]
OK, I found a couple of occurrences of references to the old titles and have now updated and pushed them into this branch.  Please have a quick look Mona, thanks.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Ok great! Please resolve the conflict, then it should be good to go :)

**Review by chris529 — COMMENTED** [review]
Look good :-)

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please fix the conflict

**Review by Tschakki — APPROVED** [review]
Thank you! :)

**Pull request merged**